modules/storage/storage-account/bicepconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+    "experimentalFeaturesEnabled": {
+      "userDefinedTypes": true,
```

Comment:
This should work once Bicep v0.17 is out.

Reply:
Ok, so we need to wait before we could merge this.

Reply:
@shenglol when are we expecting 0.17 bicep to be out ?

Reply:
@AshutoshNirkhe  Should be soon I guess from https://github.com/Azure/bicep/pull/10604

Reply:
sorry guys, rebase turned to be messy. I have ended up opening a new PR here - https://github.com/Azure/bicep-registry-modules/pull/368 Please have a look, it basically has the same changes copied over to it.